osmdroid-android/build.gradle

```diff
@@ -3,8 +3,12 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion = compileSdk
+    defaultConfig {
+        minSdkVersion 14
+    }
```

Comment:
This was the working compromise. 
I would go strait to 21, the older software/apps can stay on current version. 😈 
You can't make everyone happy, especially when the minority is 0.001 %

Reply:
But this is not your private project, but a widely used library, so tradeoffs need to be made 😉 
I think 16 is a good compromise as there is virtually no-one using devices below that and in my experience it's not much harder to support than API19 (except maybe for file-access related stuff).

Reply:
On the other hand 14 doesn't hurt now

Reply:
> But this is not your private project

😄 That's right, otherwise
* this PR is merged since days. 
* CI is running
* Android X is applied
* and maybe some files are in Kotlin

Reply:
Just to clarify and prevent misunderstandings: I'm not a maintainer of osmdroid, just someone like you contributing from time to time whenever I stumble upon a problem or have the free time to do some cleanup or to chime in on a PR review.